garden_ai/client.py

```diff
@@ -322,82 +321,6 @@ def _update_datacite(
         self.backend_client.update_doi_on_datacite(payload)
         logger.info("Update request succeeded")
 
-    def add_paper(self, title: str, doi: str, **kwargs) -> None:
```

Comment:
turned out we never actually used these two methods, so I just took them out

Reply:
🫡  Farewell